prisma/schema.prisma

```diff
@@ -73,12 +73,15 @@ model VerificationToken {
 
 // Profile schema
 model Profile {
-  id            Int     @id @default(autoincrement())
-  user          User    @relation(fields: [userId], references: [id])
-  userId        String  @unique
-  preferredName String?
-  phoneNumber   String?
-  bio           String?
+  id                 Int      @id @default(autoincrement())
+  user               User     @relation(fields: [userId], references: [id])
+  userId             String   @unique
+  dateOfBirth        String?
+  preferredName      String?
+  phoneNumber        String?
+  bio                String?
+  selectedUniversity String?
```

Comment:
This can probably just be "university"

Reply:
@ulrokx @h-pyo This has a conflict with @phill52 pull request. He added university too to the Schema